website/content/en/highlights/2021-11-12-event-trottle-transform.md

```diff
@@ -0,0 +1,52 @@
+---
+date: "2021-11-16"
+title: "Event `throttle` transform released"
+description: "A guide to using the new `throttle` transform"
+authors: ["barieom"]
+pr_numbers: []
+release: "0.18.0"
+hide_on_release_notes: false
+badges:
+  type: new feature
+---
+
+# A new `throttle` transform 
+
+
+We've released a new `throttle` transform that provides a user the ability to throttle the throughput of specific event streams.
+
+Large spikes in data volume can frequently overwhelm a service, which is especially common in log data. Previously, users lacked the necessary tooling to control throughput, such as setting quota for users and user groups utilizing Vector, which not only can cause spike in costs, but also negatively impact downstream services due to the increased load. 
+
+The [`throttle`][] transform enables you to rate limit specific subsets of your event stream to limit load on downstream services or enforce quotas on users. You can utilize the `throttle` transform to enforce rate limits on number of events and exclude events based on a [VRL condition][] to avoid dropping critical logs. 
```

Comment:
@binarylogic noted that he wasn't sure about the quota language - it was pulled in from it's use in the underlying governor library used in the transform, but if we'd prefer to retain it for separate usage in vector we should rewrite the docs and the highlight to match that expectation.